modules/swagger-codegen/src/main/resources/android/libraries/volley/apiInvoker.mustache

```diff
@@ -223,6 +223,7 @@ public class ApiInvoker {
   }
 
   public static ApiInvoker getInstance() {
+    if(INSTANCE == null) initializeInstance();
```

Comment:
@Richou Please add a space after `if` so that the code conforms to https://source.android.com/source/code-style.html